manifests/server.pp

```diff
@@ -500,6 +507,7 @@
   String $verb                              = '',
   String $cipher                            = 'AES-256-CBC',
   String $tls_cipher                        = 'TLS-DHE-RSA-WITH-AES-256-GCM-SHA384:TLS-DHE-RSA-WITH-AES-256-CBC-SHA256:TLS-DHE-RSA-WITH-AES-128-GCM-SHA256:TLS-DHE-RSA-WITH-AES-128-CBC-SHA256',
+  String $tls_version_min                   = '',
```

Comment:
Datatypes \o/

Reply:
Can you change the logic so it defaults to undef instead of empty string? That would be the current best practice.